test_programs/execution_success/databus/src/main.nr

```diff
@@ -1,4 +1,4 @@
-fn main(mut x: u32, y: call_data u32, z: call_data [u32; 4]) -> return_data u32 {
+fn main(mut x: u32, y: call_data(0) u32, z: call_data(0) [u32; 4]) -> return_data u32 {
```

Comment:
Please add a test that utilizes two different `call_data` ID's

Reply:
I'd rather wait for multiple call data to be supported in BB. I could re-use the current interface but I am not sure how Barretenberg is supposed to handle them.